exercises/practice/list-ops/.docs/instructions.append.md

```diff
@@ -0,0 +1,8 @@
+# Emacs Lisp Track specific functions
+
+* `list-empty-p` (*given a list, return if the list is empty*)
+* `list-sum` (*given a list of numbers, return the sum of all elements*)
+
+## Hints
+
+The `fold-left` and `fold-right` methods are "fold" functions, which is a concept from the functional programming world. See the Wikipedia page on folding for [general background](https://en.wikipedia.org/wiki/Fold_(higher-order_function)) and [signature/implementation hints](https://en.wikipedia.org/wiki/Fold_(higher-order_function)#Linear_folds).
```

Comment:
These should go into a separate `hints.md` file (see https://github.com/exercism/elixir/blob/main/exercises/practice/strain/.docs/hints.md for an example)

Reply:
Like this? https://github.com/exercism/emacs-lisp/pull/225/commits/86291ac7abc7afd286b8071761a06303af30fe55